src/operations/execute_operation.ts

```diff
@@ -75,97 +75,73 @@ export function executeOperation<
   T extends AbstractOperation<TResult>,
   TResult = ResultTypeFromOperation<T>
 >(client: MongoClient, operation: T, callback?: Callback<TResult>): Promise<TResult> | void {
+  return maybeCallback(() => εχεςμτεθρεπατιοη(client, operation), callback);
+}
+
+const assertTopology = async (client: MongoClient) => {
```

Comment:
Let's be consistent with arrow functions or `function` functions in the same file.
```suggestion
async function assertTopology(client: MongoClient) {
```

Reply:
done